src/main/resources/data/invoices/45723_canceled.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "07bb89be-dd4a-42f8-bbd0-a648e14bac5d",
+  "adjustments": [
+  ],
+  "adjustmentsTotal": 0.00,
+  "currency": "USD",
+  "folioInvoiceNo": "45723",
+  "invoiceDate": "2018-07-20T00:00:00Z",
+  "paymentMethod": "EFT",
+  "status": "Cancelled",
+  "source": "4a151bd6-771e-405c-9cbe-0dcd035c58c4",
+  "subTotal": 25.00,
+  "total": 25.00,
```

Comment:
PO with id="e20af8c8-b07d-47a1-9ebd-f1187e9335bd" this invoice is based on has 1 PO Line with estimated total `5960`.

Reply:
Updated. Now this invoice covers 1 resource of the 2 ordered .